src/busio.py

```diff
@@ -279,47 +280,7 @@ def __init__(self, clock, MOSI=None, MISO=None):
 
     def configure(self, baudrate=100000, polarity=0, phase=0, bits=8):
         """Update the configuration"""
-        if detector.board.any_raspberry_pi or detector.board.any_raspberry_pi_40_pin:
```

Comment:
did u mean to delete?

Reply:
Oh, we should add `any_raspberry_pi_40_pin` to https://github.com/adafruit/Adafruit_Python_PlatformDetect/blob/main/adafruit_platformdetect/board.py#L600-L624 first. It's not currently there, so it would stop working.

Reply:
Submitted PR https://github.com/adafruit/Adafruit_Python_PlatformDetect/pull/184